spec/features/application/user_can_view_application_show_page_spec.rb

```diff
@@ -9,41 +9,10 @@
     @pet_3 = create(:pet, name: "holie", age: 4, sex: "female", shelter: @shelter_1)
     @pet_4 = create(:pet , shelter:@shelter_2 )
     @application = Applicant.create!(name: "kwibe merci", address: "3478 MLK", city: "Denver", state: "CO", zip: "34526", phonenumber: "3452690876", description: "I love the pets. They will be my new babies ❤️" )
+    @application.pets << [@pet_2, @pet_1]
```

Comment:
Thanks for updating this here I forgot you could shovel objects in this manner. One line is much cleaner.